src/spline2/MultiBspline.hpp

```diff
@@ -27,10 +27,12 @@
 namespace qmcplusplus
 {
 /** container class to hold a 3D multi spline pointer and BsplineAllocator
-   * @tparam T the precision of splines
-   * @tparam ALIGN the alignment of the orbital dimention
-   * @tparam ALLOC memory allocator
-   */
+ * @tparam T the precision of splines
+ * @tparam ALIGN the alignment of the orbital dimention
+ * @tparam ALLOC memory allocator
+ *
+ * This class contains a pointer to a C object, copy and assign of this class is forbidden.
+ */
```

Comment:
This should be a class
private:
SplineType* spline_m;
public:
SplineType* getSplinePtr();

Will prevent issue like in SplineC2R.

Reply:
Good suggestion. Done. "Will prevent issue like in SplineC2R." What is the issue you refer to?

Reply:
The assignment of the extra Multipline pointer wouldn't have compiled. Takes a bit more obliviousness to take and keep an extra copy of the pointer in new code.